CHANGELOG.md

```diff
@@ -11,6 +11,12 @@
 
 - Fixed some quirk with the formatting of binary operators.
 
+### Config
```

Comment:
Could you change this to `Build tool` and put it above the bugs section? Thank you.